Workflow automation/Upgrade Pricing Tier/readme.md

```diff
@@ -0,0 +1,23 @@
+Upgrading Pricing Tier

```

Comment:
There are Azure Policy definitions already to enforce Azure Defender on a subscription. In addition to that, Azure Defender needs to be enabled as a solution on the Log Analytics workspace for some plans.